osu.Game.Rulesets.Mania/Mods/ManiaModNoLongNotes.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mania.Objects;
+using osu.Game.Rulesets.Mods;
+using osu.Framework.Graphics.Sprites;
+using System;
+using System.Collections.Generic;
+using osu.Game.Audio;
+using osu.Game.Rulesets.Mania.Beatmaps;
+using osu.Framework.Utils;
+using osu.Game.Overlays.Settings;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+
+
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModNoLongNotes : Mod, IApplicableAfterBeatmapConversion
+    {
+
+        public override string Name => "No Long Notes";
+
+        public override string Acronym => "NL";
+        
+        public override double ScoreMultiplier => 1;
+        
+        public override string Description => @"Turns all held notes into tap notes. No coordination required.";
```

Comment:
as above, "tap" notes aren't really a term that gets used i don't think

```suggestion
        public override string Description => @"Turns all held notes into normal notes. No coordination required.";
```

Reply:
How might I run inspectcode?
Also, under CONTRIBUTING.md it warns of pushing too many times to avoid burdening the CI queue. I'd gladly push your suggestion here if you'd confirm it's not a problem (otherwise I will implement all of the suggestions in this discussion and push them at once).

Reply:
for running inspectcode, see [readme](https://github.com/ppy/osu#code-analysis). if you're on a non-windows platform `InspectCode.sh` is also provided in the repository root.

as for the suggestion - it is fine to apply it locally as well and push out all required changes at once, not necessary to commit it via github.